web/layouts/shortcodes/guide-list.html

```diff
@@ -0,0 +1,14 @@
+{{ $path := default "nightly" (.Get "version") }}
+
+* [Planning for Foreman]({{ $path }}/Planning_Guide/index-foreman.html)
+* [Installing Foreman on RHEL/CentOS]({{ $path }}/Installing_Server_on_Red_Hat/index-foreman.html)
+* [Installing Smart Proxy on RHEL/CentOS]({{ $path }}/Installing_Server_on_Red_Hat/index-foreman.html)
+* [Installing Foreman on Debian/Ubuntu]({{ $path }}/Installing_Server_on_Debian/index-foreman-deb.html)
+* [Installing Smart Proxy on Debian/Ubuntu]({{ $path }}/Installing_Proxy_on_Debian/index-foreman-deb.html)
+* [Deploying Foreman on AWS (reference guide)]({{ $path }}/Deploying_on_AWS/index-foreman.html)
+* [Configuring Smart Proxies with a Load Balancer]({{ $path }}/Configuring_Load_Balancer/index-foreman.html)
+* [Provisioning Guide]({{ $path }}/Provisioning_Guide/index-foreman.html)
+* [Content Management Guide]({{ $path }}/Content_Management_Guide/index-foreman.html)
+* [Configuring Foreman to use Ansible]({{ $path }}/Configuring_Ansible/index-foreman.html)
+* [Managing Hosts Guide]({{ $path }}/Managing_Hosts/index-foreman.html)
+* [Administering Foreman]({{ $path }}/Administering_Red_Hat_Satellite/index-foreman.html)
```

Comment:
This bit is reused on the landing (index) page as well as the 404 page. And in the future we can easily include this on version page. E.g. on the 2.3 page I am going to include this with a version argument.